lib/literature/migrations/postgres/v03.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Literature.Migrations.Postgres.V03 do
+  @moduledoc """
+  Add literature_redirects table
+  """
+  use Ecto.Migration
+
+  def up(_opts) do
+    create_if_not_exists table(:literature_redirects, primary_key: false) do
+      add(:id, :binary_id, primary_key: true)
+      add(:from, :string, null: false)
+      add(:to, :string, null: false)
+      add(:type, :integer, null: false)
+
+      add(
+        :publication_id,
+        references(:literature_publications, on_delete: :nilify_all, type: :binary_id)
+      )
+
+      timestamps()
```

Comment:
@rcmonsayac can we make sure we are adding indexes for from to and publication_id

Reply:
@dgross881 added indexes